app/app.js

```diff
@@ -0,0 +1,14 @@
+const express = require("express");
+const app = express();
+const db = require("../db/connection")
+const endpointsJSON = require("../endpoints.json")
+
+// Middleware to handle JSON responses
+app.use(express.json());
+
```

Comment:
Are you using this right now? or is this boilerplate for future use? We only need app.use when we are handling a POST request.